### PR TITLE
fix: refracto settings ui

### PR DESCRIPTION
**Pull request by clegirar**
Before:

Signed-off-by: clegirar <clemntgirard@gmail.com>

### CODE REVIEW

File: js/packages/components/onboarding/CustomMode.tsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+							alignItems: 'center',
+							justifyContent: 'space-between',
+						},
+					]}
+				>
+					<TouchableOpacity
+						style={[
+							padding.medium,
+							border.radius.medium,
+							{ width: 170 * scaleSize, backgroundColor: '#EBECFF' },
```
Comment (JohnBerd) [inline js/packages/components/onboarding/CustomMode.tsx:121]: You can use width(170), not necessary to modify on this PR

File: js/packages/components/settings/Accounts.tsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+	pbDateToNum,
+	closeAccountWithProgress,
+} from '@berty-tech/store'
+
+import { ButtonSettingV2, Section } from '../shared-components'
+import {
+	selectSelectedAccount,
+	setStateOnBoardingReady,
+} from '@berty-tech/redux/reducers/ui.reducer'
+import { useDispatch, useSelector } from 'react-redux'
```
Comment (JohnBerd) [inline js/packages/components/settings/Accounts.tsx:20]: Move to the top of the file

File: js/packages/components/settings/DevTools.tsx [hunk 3]
```diff
@@ -7,7 +7,6 @@ import Long from 'long'
 import { withInAppNotification } from 'react-native-in-app-notification'
```
Comment (JohnBerd) [inline js/packages/components/settings/DevTools.tsx:7]: Imports can be reordered

**Review by JohnBerd — COMMENTED** [review]
LGTM 👍 
Great job!

File: js/packages/components/settings/Accounts.tsx (hunk 2 above)
Comment (clegirar) [reply js/packages/components/settings/Accounts.tsx:20]: I think we have to code a tool that reorder imports 👍

**Pull request merged**